docs/product-mapping.schema.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I don't like this being in the root of the docs directory as it's really a tooling thing, it's not related to the content.